sparkle/src/components/Icon.tsx

```diff
@@ -1,10 +1,25 @@
-import React, { ComponentType } from "react";
+import React, { ComponentType, CSSProperties } from "react";
+import { classNames } from "@sparkle/lib/utils";
 
-interface IconProps {
+export interface IconProps {
   IconComponent?: ComponentType<{ className?: string }>;
```

Comment:
Isn't that a bit verbose as Name? Will that mean we have to use `IconComponent={...}` anywhere we use an icon outside of the design system, right? That does not happen that often that being said...

Reply:
Roger that :+1: